src/main/java/com/epam/lab/payments/web/html/cosmetic/CardNumberAdjuster.java

```diff
@@ -0,0 +1,25 @@
+package com.epam.lab.payments.web.html.cosmetic;
+
+/**
+ * formatter for card numbers to represent them more human friendly
+ */
+public class CardNumberAdjuster {
+
+    public static String valueOf16Digits(String value) {
+
+        if (value.length() < 10) {
+            return value;
+        }
+
+        StringBuilder builder = new StringBuilder();
```

Comment:
If you want to use StringBuilder, you can do it in one pipeline and in return statement. I think it will be more concise.
Like this
`return new StringBuilder()`
`      .append(...)`
`      ....`
`      .toString()`

Reply:
indeed, thanks